app/controllers/users_controller.rb

```diff
@@ -1,4 +1,29 @@
 class UsersController < ApplicationController
+
+  def show
+    @user = User.find_by id:params[:id]
```

Comment:
space between key and value 

``
User.find_by id: params[:id]
``

Reply:
Ok a
I was edited